### DISCUSSION TITLE
Slimefun Addons

### DISCUSSION THREAD

**Discussion by Peskdt** [TownyAdvanced/TownyResources#76] [Q&A] [answered]
Does the plugin work with slimefun addons? like ExoticGarden

**Comment by LlmDl** [comment, marked as answer]
Yes see the [readme](https://github.com/TownyAdvanced/TownyResources#readme).

**Reply by Peskdt** [reply]
It works on config, but it can't be found in the surveys :C

**Reply by LlmDl** [reply]
Then file a bug ticket. I cannot read what you've put in your config which isn't surveyable.
